py/selenium/webdriver/common/action_chains.py

```diff
@@ -61,7 +66,7 @@ class ActionChains:
     another.
     """
 
-    def __init__(self, driver, duration=250, devices=None):
+    def __init__(self, driver, duration: int = 250, devices: Optional[List[any_device]] = None) -> None:
```

Comment:
why driver is not typed?

Reply:
> why driver is not typed?

I will fix this.

Reply:
Getting `Circular Import` error when tried to imported `Chrome`, or `Firefox`, or `Ie`. `Partially initialized module`

Reply:
https://stackoverflow.com/questions/39740632/python-type-hinting-without-cyclic-imports

Reply:
Thanks. I have made appropriate changes. Please check now.

Reply:
I believe the driver should have been typed as `selenium.webdriver.remote.webdriver.WebDriver`

Reply:
Yeah.. Instead of making `driver` as type of sub-classes, we can directly make `driver` of type `selenium.webdriver.remote.webdriver.WebDriver`. Thanks for pointing this out.. will make this change.